docs/en/benchmark/performance_evaluation_guide.md

```diff
@@ -82,67 +82,67 @@ The next two performance observation and analysis tools are essential tools for
 
 ### JuiceFS Stats
 
-JuiceFS `stats` is a tool for real-time statistics of JuiceFS performance metrics, similar to the `dstat` command on Linux systems, which displays changes in metrics for JuiceFS clients in real time (see [documentation](stats_watcher.md) for detailed). When the `juicefs bench` is running, create a new session and execute the following command.
+JuiceFS `stats` is a tool for real-time statistics of JuiceFS performance metrics, similar to the `dstat` command on Linux systems, which displays changes of metrics for JuiceFS clients in real time (see [documentation](stats_watcher.md) for detailed). When the command `juicefs bench` is running, create a new session and execute the following command.
```

Comment:
```suggestion
JuiceFS `stats` is a tool for real-time statistics of JuiceFS performance metrics, similar to the `dstat` command on Linux systems. It can display changes of metrics for JuiceFS clients in real-time (see [documentation](stats_watcher.md) for details). For this,  create a new session and execute the following command when the command `juicefs bench` is running, 
```